modules/popup/css/lib/popup.scss

```diff
@@ -1,203 +1,120 @@
-    }
+  to {
+    opacity: 1;
+    transform: translate(0);
   }
+}
 
-  &-left {
-    margin-right: $wdc-spacing-s;
-
-    &:before,
-    &:after {
-      top: $_wdc-popup-arrow-offset;
-      left: 100%;
-    }
-
-    &:before {
-      margin-top: -($_wdc-popup-arrow-size + 1px);
-      border-left-color: $_wdc-popup-border-color;
-    }
+@keyframes popupAnimationLeftCenter {
+  from {
+    opacity: 0;
+    transform: translate(50%, 0);
+  }
 
-    &:after {
-      margin-top: -$_wdc-popup-arrow-size;
-      border-left-color: $_wdc-popup-bg-color;
-    }
+  to {
+    opacity: 1;
+    transform: translate(0);
   }
+}
 
-  &-top {
-    margin-bottom: $wdc-spacing-s;
+@keyframes popupAnimationBottomCenter {
+  from {
+    opacity: 0;
+    transform: translate(0, 50%);
+  }
 
-    &:before,
-    &:after {
-      top: 100%;
-      left: $_wdc-popup-arrow-offset;
-    }
+  to {
+    opacity: 1;
+    transform: translate(0);
+  }
+}
 
-    &:before {
-      margin-left: -($_wdc-popup-arrow-size + 1px);
-      border-top-color: $_wdc-popup-border-color;
-    }
+.wdc-popup {
+  position: relative;
+  background-color: $_wdc-popup-bg-color;
+  padding: $wdc-spacing-s;
+  border-radius: 3px;
+  border: 1px solid $_wdc-popup-border-color;
+  position: relative;
+  width: max-content;
+  max-width: $_wdc-popup-max-width;
+  animation-duration: 150ms;
+  animation-timing-function: ease-out;
+  position: absolute;
+
+  .wdc-popup-close {
+    position: absolute;
+    right: $wdc-spacing-xs;
+    top: $wdc-spacing-xs;
+  }
 
-    &:after {
-      margin-left: -$_wdc-popup-arrow-size;
-      border-top-color: $_wdc-popup-bg-color;
-    }
+  .wdc-popup-title {
+    @include wdc-type-h3();
+    margin-bottom: $wdc-spacing-m;
   }
 
-  &-bottom {
-    margin-top: $wdc-spacing-s;
+  .wdc-popup-body {
+    @include wdc-type-body();
+  }
 
-    &:before,
-    &:after {
-      bottom: 100%;
-      left: $_wdc-popup-arrow-offset;
-    }
+  &-animation-origin-top-center {
+    animation-name: popupAnimationTopCenter;
+    transform-origin: top center;
+  }
 
-    &:before {
-      margin-left: -($_wdc-popup-arrow-size + 1px);
-      border-bottom-color: $_wdc-popup-border-color;
-    }
+  &-animation-origin-right-center {
+    animation-name: popupAnimationRightCenter;
+    transform-origin: right center;
+  }
 
-    &:after {
-      margin-left: -$_wdc-popup-arrow-size;
-      border-bottom-color: $_wdc-popup-bg-color;
-    }
+  &-animation-origin-bottom-center {
+    animation-name: popupAnimationBottomCenter;
+    transform-origin: bottom center;
   }
 
-  .wdc-popup-title {
-    font-weight: 500;
-    margin-bottom: $wdc-spacing-xxs;
+  &-animation-origin-left-center {
+    animation-name: popupAnimationLeftCenter;
+    transform-origin: left center;
   }
 
-  p {
-    margin-top: 0;
-    margin-bottom: 0;
+  // Fallback for IE11
+  @media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) {
```

Comment:
We don't use `width: max-content` in card, which means we'll be able to drop these.